_posts/2024-06-10-first-cockpit-files-release.md

```diff
@@ -0,0 +1,27 @@
+---
+title: First Cockpit-Files release
+author: jelle
+date: 2024-06-10
+category: release
+tags: cockpit files manager browser
+---
+
+Last Friday we made the first release of cockpit-files, a new plugin for Cockpit to manage files on your server. Development on this new plugin was started by our GSoC student [Mahmoud Hamdy](https://github.com/MahmoudHamdy02) with the intention to replace existing [cockpit-navigator](https://github.com/45drives/cockpit-navigator/) plugin from 45Drives with a modern, automated tested and Cockpit styled plugin.
+
+This first release includes download, upload, copy & paste, renaming of files & directories, directory creation and editing of permissions. In future releases we intend to add a basic file editor for configuration files, cut & paste, SELinux, ACL's, symbolic link creation support and more.
+
+## Card view
+
+![Cockpit Files overview](/images/cockpit-files-1-overview.png)
+
+## List view
+
+![Cockpit Files list view](/images/cockpit-files-1-listview.png)
```

Comment:
In general I think the screenshots are too large.  We might fake the contents of a "typical home directory" or other folder with 6-10 items in it which will show off all of the features (a folder or two, a wide range of file sizes, dates, file types) in a smaller amount of space.  An endless sea of folders with 'techy' names isn't too impressive looking.

Maybe also different folders/content for the icon vs list views to highlight the relative strengths of each?